### DISCUSSION TITLE
zram-config nowhere

### DISCUSSION THREAD

**Discussion by IdolIcon** [SpiralLinux/SpiralLinux-project#485] [Q&A]
I wanted to enable zRAM the same way I do it in other Debian based distros (installing zram-config, the only way I know to do it), but zram-config doesn't appear in Synaptic like in the other distros.

Is something wrong in my Spiral Xfce (ext4) system? Can I install it in a different way?

**Comment by geckolinux** [comment]
Hi there, it's installed by default in SpiralLinux, and the config file is in `/etc/default/zramswap` .
There is no `zram-config` package for Debian, it's called [zram-tools](https://packages.debian.org/bookworm/zram-tools) .

**Reply by IdolIcon** [reply]
Maybe I installed 'zram-config' on distros prior to the current Debian. I'm not sure now.
I'll try 'zram-tools'. Thank you very much for your info.
